modules/rules_java/5.0.0/MODULE.bazel

```
@@ -43,3 +39,10 @@ use_repo(toolchains, "local_jdk")
 
 # Dev dependencies
 bazel_dep(name = "rules_pkg", version = "0.5.1", dev_dependency = True)
+
+# Register toolchains
+register_toolchains(
+        "//toolchains:all",
+        "@local_jdk//:runtime_toolchain_definition",
+)
+[register_toolchains(toolchain) for toolchain in REMOTE_JAVA_TOOLCHAINS]
```

In case I'm not messing this up again, the following would be a bit more idiomatic:
```suggestion
register_toolchains(
        "//toolchains:all",
        "@local_jdk//:runtime_toolchain_definition",
        *REMOTE_JAVA_TOOLCHAINS,
)
```